week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -10,17 +10,33 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
-
+  
+  constructor() {
+    // this.currentTotal = 0;
+    this.currentTotal = [];
+  }
+  addItem(item) {
+  
+  this.currentTotal.push(item);
+  }
   cartContains() {
     // Use console.log() to output everything contained in your cart
+  console.log(this.currentTotal);
+  
+    return this.currentTotal;
   }
 }
 
 let myCart = new ShoppingCart(); // Creates an empty shopping cart
+myCart.addItem("Toilet Roll");
+myCart.addItem("Pasta");
+myCart.addItem("Eggs");
+console.log(myCart.cartContains());
```

Comment:
you don't need to console log the method, it has console log inside, you just need to call it